optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/roster/RosterRestServiceImpl.java

```diff
@@ -339,12 +340,14 @@ public void updateShiftsOfRoster(Roster newRoster) {
 
     @Override
     @Transactional
-    public void publishAndProvision(Integer tenantId) {
+    public List<LocalDate> publishAndProvision(Integer tenantId) {
```

Comment:
Let's use a strongly typed approach. Here we are hardcoding that that is a List of 2 values, and the elements are not self-explenatory.

Let's return a new class PublishResult with has 2 LocalDate fields.

Reply:
Done